lookout/style/format/rule_stat.py

```diff
@@ -44,16 +46,45 @@ def print_rule_table(rule_stat):
         print(new_line)
 
 
+def pretty_rule(rule: Rule, feature_names: Sequence[str], class_names: Sequence[str]) -> str:
```

Comment:
Format of a function name requires a verb. E.g.: `pretty_format_rule` is Pythonic.